test/common/filter/tcp_proxy_test.cc

```diff
@@ -378,7 +378,8 @@ class TcpProxyTest : public testing::Test {
       conn_info.connection_ = upstream_connection_;
       conn_info.host_description_.reset(
           new Upstream::HostImpl(cluster_manager_.thread_local_cluster_.cluster_.info_, "",
-                                 Network::Utility::resolveUrl("tcp://127.0.0.1:80"), false, 1, ""));
```

Comment:
Any reason to not use makeTestHost in the few other places where you didn't already?